DigiMeSDK/Core/Classes/Security/DMEJWTUtility.swift

```diff
@@ -0,0 +1,446 @@
+//
+//  DMEJWTUtility.swift
+//  DigiMeSDK
+//
+//  Created on 04/12/2019.
+//  Copyright © 2019 digi.me Limited. All rights reserved.
+//
+
+import CryptorRSA
+import Foundation
+import SwiftJWT
+
+@objcMembers
+public class DMEJWTUtility: NSObject {
+   
+    // Default JWT header
+    static var header = Header(typ: "JWT",
```

Comment:
Looking at how JWT uses this - it retains the instance of this internally, which means you could end up with multiple instances of JWT holding reference to the same instance. Given that all we need do is pass this to JWT during initialization, we can just create it on the fly when needed, so I'd change this to:

```swift
var header {
    Header(...)
}
```